src/commands/communities/setcommunityfilters.js

```diff
@@ -69,13 +67,7 @@ class SetFilters extends Command {
 			const confirm = await getConfirmationMessage(message, "Are you sure you want to set your community filters to this?")
 			if (!confirm) return message.channel.send("Setting of trusted communities has been cancelled")
 
-			const request = await fetch(`${this.client.config.apiurl}/communities/setconfig`, {
-				method: "POST",
-				body: JSON.stringify({
-					trustedCommunities: trustedCommunities
-				}),
-				headers: { "apikey": config.apikey, "content-type": "application/json" }
-			}).then(r => r.json())
+			const request = await this.client.fagc.communities.setConfig({trustedCommunities}, {apikey: config.apikey})
```

Comment:
Not sure why apikey needs to be provided here when everything in the wrapper implies the apikey is set during instantiation.

Reply:
Wrong implementation on the wrapper side. Fixed now, just had to make it optional.